documents/operating.md

```diff
@@ -52,14 +52,223 @@ SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
 SECTION 5: 🗣🗣🗣 Governing Process 🗣🗣🗣
 ---------------------------------------------------------------------
-- [ ] Types of decision
-- [ ] Records and documentation
-- [ ] Membership
-- [ ] Mechanisms for deliberation
-  - [ ] ... On GitHub
-  - [ ] ... On Slack
-- [ ] Use of emoji
-- [ ] Meetings
+- [x] Types of decision
+- [x] Records and documentation
+- [x] Mechanisms for deliberation
+  - [x] ... On GitHub
+  - [x] ... On Slack
+  - [x] ... Plotting Sessions
+- [x] Use of emoji
+
+### Schedules and Expectations
+All meaningful organizational decisions will happen in monthly Plotting
+Sessions, which are to be scheduled and announced by the Overmind on
+April 1st of every year.  The schedule can be updated as needed, but
+a notice of 10 days must be given before any Plotting Session can be
+held.
+
+April 1st marks the beginning of the governing year, and officer elections
+are to be held during the month of April.
+
+
+### Deliberation Processes
+#### Plotting Session
+Plotting Sessions are agenda driven and require a Quorum.  They
+are led by the chair (as determined by the chain of command).
+
+The Hive Mind will use Robert's Rules of Order to run Plotting Sessions
+in an orderly and Robertly fashion.
+
+When possible, video should be enabled for any plotting session.  Votes should be
+cast using physical emoji 👍, 👎, and 😐 (or any skin tone variation) depicted
+on signs attached to sticks and held up to indicate votes of yes, no, and abstain
+respectively.  Members can also vote by raising their hands, but this will
+be considered lame.
+
+A role call vote can take place instead if the Secretary of Lore or the meeting
+chair deems it necessary in order to properly tally a vote.
+
+
+#### GitHub
+Decisions made on GitHub are either interim decisions that must
+culminate in a Plotting Session to finalize, or they are decisions
+related to BIFFUD's GitHub process and documentation policy.
+
+Almost all decisions involve GitHub at some level.
```

Comment:
Are folks OK with this?  To me this kills two birds with one stone in that the process and outcome is documented and will result in the changes we want, but if we are worried it will be too much overhead to do properly that's understandable.

For example:
- Adding a project vote would involve filling out a template file and PRing its addition to the repo
- Updating a project vote would involve changing that project's repo file and PRing the changes
- Adding an overlord would involve adding the overlord's name to our overlords list and PRing the change

Point being that none of these things are intended to be super involved, and the PR is simply a way to register the request in an auditable way and log the outcome.

I'm totally fine with removing all form of deliberation from GitHub outside of votes, however, if we are worried that would get unwieldy (e.g. questions and answers wouldn't happen in GitHub)